app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditSshKeyItems.kt

```diff
@@ -0,0 +1,127 @@
+package com.x8bit.bitwarden.ui.vault.feature.addedit
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyListScope
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.x8bit.bitwarden.R
+import com.x8bit.bitwarden.ui.platform.components.field.BitwardenPasswordField
+import com.x8bit.bitwarden.ui.platform.components.field.BitwardenTextField
+import com.x8bit.bitwarden.ui.platform.theme.BitwardenTheme
+import com.x8bit.bitwarden.ui.vault.feature.addedit.handlers.VaultAddEditCommonHandlers
+import com.x8bit.bitwarden.ui.vault.feature.addedit.handlers.VaultAddEditSshKeyTypeHandlers
+
+/**
+ * The UI for adding and editing a SSH key cipher.
+ */
+fun LazyListScope.vaultAddEditSshKeyItems(
+    commonState: VaultAddEditState.ViewState.Content.Common,
+    sshKeyState: VaultAddEditState.ViewState.Content.ItemType.SshKey,
+    commonTypeHandlers: VaultAddEditCommonHandlers,
+    sshKeyTypeHandlers: VaultAddEditSshKeyTypeHandlers,
+) {
+    item {
+        Spacer(modifier = Modifier.height(8.dp))
+        BitwardenTextField(
+            label = stringResource(id = R.string.name),
+            value = commonState.name,
+            onValueChange = commonTypeHandlers.onNameTextChange,
+            modifier = Modifier
+                .testTag("ItemNameEntry")
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp),
```

Comment:
I am wondering if we should be using the `standardHorizontalPadding()` modifier for all these changes.

Reply:
Probably. Much of this is copied from the other `*Items` screens. I'll update the modifiers here and update the other screen in a separate PR.